### ISSUE TITLE
PyInstaller with pyzmq not working.

### ISSUE THREAD

**Issue by dukesteen** [zeromq/pyzmq#1501]
Hi everyone, I have a weird issue where I can run my script from PyCharm just fine, but when I use pyinstaller to make a single file executable, it gives me this error:

```
Traceback (most recent call last):
  File "main.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "zmq\__init__.py", line 19, in <module>
  File "zmq\__init__.py", line 13, in _delvewheel_init_patch_0_0_9
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Duke\\AppData\\Local\\Temp\\_MEI286442\\pyzmq.libs\\.load_order'
[15396] Failed to execute script main
```

Is there a known fix for this?
Currently using `pyzmq==22.0.2` and `pyinstaller==4.2`

Thanks in advance for any replies!

**Comment by dukesteen** [comment]
https://stackoverflow.com/questions/66054625/pyinstaller-error-running-script-with-pyzmq-dependency
Here is another person with the same question posted to stackoverflow, no answer there yet either.

**Comment by dukesteen** [comment]
Fix: Downgrade to 21.0.0 using `pip install pyzmq==21.0.0`
